app/assets/javascripts/lib/ui/content/annotations/elide.js

```diff
@@ -8,8 +8,17 @@ delegate(document, '.annotate.elide', 'click', e => {
   let annotationId = e.target.dataset.annotationId;
   let elisions = document.querySelectorAll(`.annotate.elided[data-annotation-id="${annotationId}"]`);
 
-  e.target.classList.toggle('revealed')
+  e.target.classList.toggle('revealed');
+  if (e.target.classList.contains('revealed')){
```

Comment:
Would this always be true since it was set to `revealed` in the previous line?

Reply:
Nope, actually! 

I love toggle: if the class is present, toggle removes it. If the class is NOT present, toggle adds it. This event handler is called every time you click the button, so it has to handle BOTH when you want to reveal the elided text and when you want to hide it. Since there isn't a toggle method for the other stuff, I had to make a separate if/else.